src/main/scala/org/broadinstitute/dsde/rawls/model/VaultModel.scala

```diff
@@ -0,0 +1,17 @@
+package org.broadinstitute.dsde.rawls.model
```

Comment:
Yeah, and since this isn't really workspace-related, it seemed like a reasonable point of separation.